app/models/episode_import.rb

```diff
@@ -49,6 +49,8 @@ def import
     set_audio_metadata!
     update!(status: PodcastImport::AUDIO_SAVED)
 
+    lookup_and_assign_episode!
```

Comment:
new method to assign already existing episodes when attempting to create another import that has the same/existing episodes on it.